src/routes/charger-routes.js

```diff
@@ -1,13 +1,15 @@
 const {
-  getCharger,
-  getChargers,
-  createCharger,
-  updateCharger,
-  deleteCharger,
-  searchChargersLocation,
```

Comment:
This is needed for the search location page, are we planning to handle this differently? The controller action still exists but now has no routing

Reply:
deleted by mistake. will change now